stable/redis-ha/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+name: redis-ha
+home: http://redis.io/
+engine: gotpl
+keywords:
+- redis
+- keyvalue
+- database
+version: 0.2.0
+description: Highly available Redis cluster with multiple sentinels and standbys.
+maintainers:
+  - email: ikaboubi@gmail.com
+    name: KABOUBI
+details:
+  This package provides a highly available Redis cluster with multiple sentinels and standbys.
+  Note the `redis-master` pod is used for bootstrapping only and can be deleted once
+  the cluster is up and running.
```

Comment:
I wouldn't consider this a blocker, but it'd be cool if we could get  a `Job` that:

* waits for the master to PING
* waits for the slaves to follow the master
* waits for the sentinels to form a cluster
* kills the master node

It'd be a nice stopgap until we get a proper redis operator.